<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget {
	background-image: url(:/new/prefix1/MainScreenApp (1).jpg);
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnEspresso">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>81</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4E342E;  /* Dark Coffee Brown */
    color: white;
    font-size: 16px;
    font-weight: bold;
    border-radius: 8px;
    padding: 8px 16px;
    border: 2px solid #3E2723;  /* Slightly darker brown for depth */
    transition: all 0.3s ease-in-out;
}

/* Hover effect */
QPushButton:hover {
    background-color: #6D4C41;  /* Slightly lighter brown */
    transform: scale(1.05);
}

/* Clicked effect */
QPushButton:pressed {
    background-color: #3E2723;  /* Dark brown when pressed */
    transform: scale(0.95);
}

/* Disable effect */
QPushButton:disabled {
    background-color: #A1887F;  /* Light brown when disabled */
    color: #ECEFF1;
    border: 2px solid #8D6E63;
}
</string>
    </property>
    <property name="text">
     <string>Espresso</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCappuccino">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>158</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4E342E;  /* Dark Coffee Brown */
    color: white;
    font-size: 16px;
    font-weight: bold;
    border-radius: 8px;
    padding: 8px 16px;
    border: 2px solid #3E2723;  /* Slightly darker brown for depth */
    transition: all 0.3s ease-in-out;
}

/* Hover effect */
QPushButton:hover {
    background-color: #6D4C41;  /* Slightly lighter brown */
    transform: scale(1.05);
}

/* Clicked effect */
QPushButton:pressed {
    background-color: #3E2723;  /* Dark brown when pressed */
    transform: scale(0.95);
}

/* Disable effect */
QPushButton:disabled {
    background-color: #A1887F;  /* Light brown when disabled */
    color: #ECEFF1;
    border: 2px solid #8D6E63;
}
</string>
    </property>
    <property name="text">
     <string>Cappuccino</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLatte">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>126</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4E342E;  /* Dark Coffee Brown */
    color: white;
    font-size: 16px;
    font-weight: bold;
    border-radius: 8px;
    padding: 8px 16px;
    border: 2px solid #3E2723;  /* Slightly darker brown for depth */
    transition: all 0.3s ease-in-out;
}

/* Hover effect */
QPushButton:hover {
    background-color: #6D4C41;  /* Slightly lighter brown */
    transform: scale(1.05);
}

/* Clicked effect */
QPushButton:pressed {
    background-color: #3E2723;  /* Dark brown when pressed */
    transform: scale(0.95);
}

/* Disable effect */
QPushButton:disabled {
    background-color: #A1887F;  /* Light brown when disabled */
    color: #ECEFF1;
    border: 2px solid #8D6E63;
}
</string>
    </property>
    <property name="text">
     <string>Latte</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddFunds">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>161</width>
      <height>86</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnAddFunds {
    background: qradialgradient(cx:0.5, cy:0.5, radius:0.8,
                stop:0 #FFD700, stop:0.5 #FFC107, stop:1 #B8860B); /* Gold gradient */
    color: black;
    font-size: 18px;
    font-weight: bold;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    border-radius: 50px;  /* Circular button */
    padding: 15px 20px;
    border: 3px solid #DAA520; /* Golden border */
    min-width: 100px;
    min-height: 50px;
    text-align: center;
    image: url(:/icons/coin.png);  /* Add a coin icon */
    background-repeat: no-repeat;
    background-position: left center;
    padding-left: 35px; /* Space for icon */
    transition: all 0.3s ease-in-out;
}

QPushButton#btnAddFunds:hover {
    background: qradialgradient(cx:0.5, cy:0.5, radius:0.8,
                stop:0 #FFC107, stop:0.5 #FFD700, stop:1 #FFD700); /* Brighter gold */
    border: 3px solid #FFBF00;
}

QPushButton#btnAddFunds:pressed {
    background: qradialgradient(cx:0.5, cy:0.5, radius:0.8,
                stop:0 #B8860B, stop:0.5 #DAA520, stop:1 #FFD700); /* Darker gold */
    border: 3px solid #8B7500;
    transform: scale(0.97);  /* Click effect */
}
</string>
    </property>
    <property name="text">
     <string>Add Funds</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRefill">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>520</y>
      <width>161</width>
      <height>86</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnRefill {
    background: qradialgradient(cx:0.5, cy:0.5, radius:0.8,
                stop:0 #00BFFF, stop:0.5 #1E90FF, stop:1 #4169E1); /* Blue water gradient */
    color: white;
    font-size: 18px;
    font-weight: bold;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    border-radius: 25px;  /* Rounded button */
    padding: 15px 20px;
    border: 3px solid #4682B4; /* Blue border */
    min-width: 100px;
    min-height: 50px;
    text-align: center;
    image: url(:/icons/water-drop.png);  /* Add a water drop icon */
    background-repeat: no-repeat;
    background-position: left center;
    padding-left: 35px; /* Space for icon */
    transition: all 0.3s ease-in-out;
}

QPushButton#btnRefill:hover {
    background: qradialgradient(cx:0.5, cy:0.5, radius:0.8,
                stop:0 #1E90FF, stop:0.5 #00BFFF, stop:1 #87CEEB); /* Lighter blue */
    border: 3px solid #00BFFF;
}

QPushButton#btnRefill:pressed {
    background: qradialgradient(cx:0.5, cy:0.5, radius:0.8,
                stop:0 #4682B4, stop:0.5 #1E90FF, stop:1 #00BFFF); /* Darker blue */
    border: 3px solid #1E90FF;
    transform: scale(0.97);  /* Click effect */
}
</string>
    </property>
    <property name="text">
     <string>Refill</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPerformMaintenance">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>110</y>
      <width>94</width>
      <height>94</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnPerformMaintenance {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                stop:0 #4A4A4A, stop:0.5 #666666, stop:1 #8A8A8A); /* Metallic look */
    color: white;
    font-size: 12px;
    font-weight: bold;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    border-radius: 35px;  /* More rounded */
    padding: 8px;
    border: 4px solid #333333; /* Dark metallic border */
    min-width: 70px;
    min-height: 70px;
    max-width: 70px;
    max-height: 70px;
    text-align: center;
    background-repeat: no-repeat;
    background-position: center;
    transition: all 0.2s ease-in-out;
    border-image: url(:/new/prefix1/OIP (1).jpeg); /* Correct way */
}

QPushButton#btnPerformMaintenance:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                stop:0 #666666, stop:0.5 #888888, stop:1 #AAAAAA); /* Brighter metallic */
    border: 4px solid #666666;
}

QPushButton#btnPerformMaintenance:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                stop:0 #2E2E2E, stop:0.5 #4A4A4A, stop:1 #666666); /* Darker when pressed */
    border: 4px solid #4A4A4A;
    transform: scale(0.92);  /* Press animation */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowLogs">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>210</y>
      <width>126</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnShowLogs {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                stop:0 #3E3E3E, stop:0.5 #5A5A5A, stop:1 #7A7A7A); /* Dark gray metallic */
    color: white;
    font-size: 14px;  /* Adjusted for better readability */
    font-weight: bold;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    border-radius: 18px;
    padding: 10px;
    border: 3px solid #2A2A2A; /* Darker contrast border */
    min-width: 100px;
    min-height: 45px;
    max-width: 110px;
    max-height: 50px;
    text-align: center;
    transition: all 0.2s ease-in-out;
}

QPushButton#btnShowLogs:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                stop:0 #5A5A5A, stop:0.5 #7A7A7A, stop:1 #A0A0A0); /* Lighter metallic */
    border: 3px solid #5A5A5A;
}

QPushButton#btnShowLogs:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                stop:0 #2A2A2A, stop:0.5 #3E3E3E, stop:1 #5A5A5A); /* Darker pressed effect */
    border: 3px solid #3E3E3E;
    transform: scale(0.95); /* Subtle press animation */
}
</string>
    </property>
    <property name="text">
     <string>Show Logs</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textLogViewer">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>300</y>
      <width>274</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit#textLogViewer {
    background-color: rgba(20, 20, 20, 180); /* Dark semi-transparent background */
    color: white; /* White text for readability */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 12px;
    border: 2px solid rgba(255, 255, 255, 80); /* Subtle white border */
    border-radius: 10px; /* Rounded corners */
    padding: 10px;
    min-width: 250px;
    min-height: 120px;
    max-height: 180px;
    text-align: left;
}
</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textBrewingSteps">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textMenuRecources">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>311</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit#textMenuRecources {
    background-color: rgba(20, 14, 10, 200); /* Darker coffee brown background */
    color: #FFD700;  /* Soft golden text for a premium feel */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 18px;
    font-weight: bold;
    border: 2px solid #D2B48C;  /* Light brown border */
    border-radius: 12px;
    padding: 12px;
    text-align: left;
    text-shadow: 1px 1px 3px rgba(0, 0, 0, 0.7);  /* Subtle text shadow */
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCoffeeCup">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>141</width>
      <height>172</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>textLogViewer</zorder>
   <zorder>btnEspresso</zorder>
   <zorder>btnCappuccino</zorder>
   <zorder>btnLatte</zorder>
   <zorder>btnAddFunds</zorder>
   <zorder>btnRefill</zorder>
   <zorder>btnPerformMaintenance</zorder>
   <zorder>btnShowLogs</zorder>
   <zorder>textBrewingSteps</zorder>
   <zorder>textMenuRecources</zorder>
   <zorder>lblCoffeeCup</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
